### PR TITLE
PRODENG-291 Add script to add tags for older PMD versions.

### PR DESCRIPTION
**Pull request by tpage-alfresco**

### CODE REVIEW

File: update_pmd.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
```
Comment (dsibilio) [inline update_pmd.sh:3]:
```suggestion

# Check if at least one tag has been provided
if [ $# -eq 0 ]; then
    echo "Error: At least one tag is required."
    echo "Usage: update_pmd.sh <tag1> <tag2> ... <tagN>"
    exit 1
fi

```

To make usage very clear even for people that don't like to read 🙈

Comment (tpage-alfresco) [reply update_pmd.sh:3]: Ok, nice improvement. I was thinking that if someone provides zero versions then the script works perfectly by creating zero tags :sweat_smile:

**Review by dsibilio — APPROVED** [review]

File: update_pmd.sh (hunk 1 above)
Comment (dsibilio) [reply update_pmd.sh:3]: Successfully failed ✅

**Review by mpichura — APPROVED** [review]
LGTM

File: update_pmd.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+echo $@
+
+# Get the current date.
+today=$(date +"%Y%m%d")
+
+# Loop through all tags provided
+for tag in "$@"
+do
+    git tag "${tag}_${today}"
+    git push origin "$tag_$today"
```
Comment (gionn) [inline update_pmd.sh]:
```suggestion
    git tag "${tag}_${today}"
    git push origin "${tag}_${today}"
```

**Review by gionn — APPROVED** [review]

**Review by krdabrowski — APPROVED** [review]

File: update_pmd.sh (hunk 2 above)
Comment (tpage-alfresco) [reply update_pmd.sh]: This was the one bit of the script I didn't test, and it would have been broken. Thanks!

**Pull request merged**